fastlane/Fastfile

```diff
@@ -225,41 +227,89 @@ end
       ios_lint_localizations(input_dir: 'Simplenote', allow_retry: true)
       ios_bump_version_beta() unless ios_current_branch_is_hotfix
     end
-    ios_final_tag(options)
```

Comment:
@loremattei as I was working on the Simplenote macOS version of the API build trigger, I noticed this diff removes the call that makes the final tag.

Is this intentional? Sometimes new code makes it into the final build since a the latest beta build / tag.

Reply:
Yes, it's intentional. The idea is to move the final tag _after_ the build has been deployed to App Store and GH so that we won't have to move the tag if any error pops up during the final build. 
Basically, with the new flow, the CI job creates a draft GH release on the HEAD commit of the release branch. When the release manager publishes the draft, GitHub automatically adds the proper tag on the git repository.

Reply:
> When the release manager publishes the draft, GitHub automatically adds the proper tag on the git repository.

Wait what?! 🤯 This is cool (a bit "magical") but cool 👍 

Thank you for the explanation 🙇